docs/src/_includes/layouts/base.html

```diff
@@ -19,19 +31,19 @@
 
     <!-- social sharing previews. Make sure every page template has title, desc, and cover url -->
     <!-- Open graph -->
-    <meta property="og:title" content="{{ title }}">
-    <meta property="og:description" content="{{ desc }}">
-    <meta property="og:image" content="{{ cover }}" />
-    <meta property="og:image:alt" content="{{ coverAlt }}">
-    <meta property="og:locale" content="en_US">
+    <meta property="og:title" content="{{ page_title }}">
+    <meta property="og:description" content="{{ page_desc }}">
+    <meta property="og:image" content="{{ cover_image }}" />
+    <meta property="og:image:alt" content="{{ cover_image_alt }}">
+    <meta property="og:locale" content="{{ site.locale | replace(" -", "_" ) }}">
```

Comment:
```suggestion
    <meta property="og:locale" content="{{ site.locale | replace("-", "_" ) }}">
```

There is an extra space, so this results in unchanged `en-US`